packages/tracing-processing/src/models/tracing.ts

```diff
@@ -15,7 +15,6 @@ export const TracingEnriched = z.object({
   status: z.coerce.number(),
```

Comment:
both consumer and producer may load data on the same purpose, but with the current schema we are not able to tell them apart.
Let's add a new field and a new table column`submitterId` which will contain the id of the tenant who submitted the tracing.